src/logic/girard.lean

```diff
@@ -0,0 +1,45 @@
+/-
+Copyright (c) 2021 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro
+-/
+import data.set.basic
+
+/-!
+# Girard's paradox
+
+Girard's paradox is a proof that `Type : Type` entails a contradiction. We can't say this directly
+in lean because `Type : Type 1` and it's not possible to give `Type` a different type via an axiom,
```

Comment:
Is there a usual way of capitalising Lean in mathlib?

Reply:
Based on a bit of grep it seems that `lean` shows up in comments only about 10-20 times while `Lean` shows up 210 times. I'll change it.